### PR TITLE
Add docs regarding trace

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
Add docs regarding trace

## Motivation and Context
Fix doc updates

## Regression
No

## How Has This Been Tested?
Follow the docs as is.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/config/README.md [hunk 1]
```diff
@@ -174,16 +174,93 @@ minio server /data
...
+HTTP tracing can be enabled by using [`mc admin trace`](https://github.com/minio/mc/blob/master/docs/minio-admin-complete-guide.md#command-trace---display-minio-server-http-trace) command.
 
 Example:
-
 ```sh
-export MINIO_HTTP_TRACE=/var/log/minio.log
 minio server /data
 ```
 
+Default trace is succint only to indicate the API operations being called and the HTTP response status.
```
Comment (poornas) [inline docs/config/README.md]: succint->succinct

**Review by poornas — CHANGES_REQUESTED** [review]

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#e3bb3c57-cb43-458e-9d77-913366b06b86) |
| mint-dist-xl.sh | :x: [more...](#9b83117e-7088-4e94-ae7a-2f90b45e852f) |
<a name="e3bb3c57-cb43-458e-9d77-913366b06b86"></a>
### 7808-9d424e9/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.54:31733
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp b9fef1165d52:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 2 seconds
(2/14) Running aws-sdk-java tests ... done in 2 seconds
(3/14) Running aws-sdk-php tests ... done in 50 seconds
(4/14) Running aws-sdk-ruby tests ... done in 13 seconds
(5/14) Running awscli tests ... done in 1 minutes and 16 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... FAILED in 1 minutes and 47 seconds
{
  "name": "mc",
  "duration": "104172",
  "function": "test_put_object_with_storage_class",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-8375 --quiet --no-color cp --storage-class REDUCED_REDUNDANCY /mint/data/datafile-1-MB myminio/mc-test-bucket-26035/mc-test-object-17676  >>> \n`/mint/data/datafile-1-MB` -> `myminio/mc-test-bucket-26035/mc-test-object-17676`\nmc: <ERROR> Failed to copy `/mint/data/datafile-1-MB`. Please reduce your request\nmc: <ERROR> Session safely terminated. To resume session `mc session resume RkZumwzJ`"
}

Executed 6 out of 14 tests successfully.
```
<a name="9b83117e-7088-4e94-ae7a-2f90b45e852f"></a>
### 7808-9d424e9/mint-dist-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.54:30906
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 484c6f244e46:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 0 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 43 seconds
(4/14) Running aws-sdk-ruby tests ... done in 4 seconds
(5/14) Running awscli tests ... done in 2 minutes and 1 seconds
(6/14) Running healthcheck tests ... done in 0 seconds
(7/14) Running mc tests ... done in 58 seconds
(8/14) Running minio-dotnet tests ... done in 35 seconds
(9/14) Running minio-go tests ... done in 51 seconds
(10/14) Running minio-java tests ... FAILED in 46 seconds
{
  "name": "minio-java",
  "function": "removeIncompleteUpload(String bucketName, String objectName)",
  "duration": 739,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code=SlowDown, message=Please reduce your request, bucketName=minio-java-test-20tp4ej, objectName=minio-java-test-fc0bjt, resource=/minio-java-test-20tp4ej/minio-java-test-fc0bjt, requestId=15A9AAFF9CB11BA1, hostId=db50d7b7-9434-4135-bb54-26a0dd78585e)\nrequest={method=DELETE, url=http://72.28.97.54:30906/minio-java-test-20tp4ej/minio-java-test-fc0bjt?uploadId=80409845-b207-4687-b45c-191b3a429de6, headers=Host: 72.28.97.54:30906\nUser-Agent: MinIO (amd64; amd64) minio-java/dev\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20190619T175240Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20190619/us-east-1/s3/aws4_request, SignedHeaders=host;x-amz-content-sha256;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=503, headers=Accept-Ranges: bytes\nContent-Length: 361\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nRetry-After: 120\nServer: MinIO/DEVELOPMENT.2019-06-19T17-33-35Z\nVary: Origin\nX-Amz-Request-Id: 15A9AAFF9CB11BA1\nX-Minio-Deployment-Id: db50d7b7-9434-4135-bb54-26a0dd78585e\nX-Xss-Protection: 1; mode=block\nDate: Wed, 19 Jun 2019 17:52:41 GMT\n}\n >>> [io.minio.MinioClient.executeReq(MinioClient.java:1204), io.minio.MinioClient.execute(MinioClient.java:1070), io.minio.MinioClient.executeDelete(MinioClient.java:1347), io.minio.MinioClient.abortMultipartUpload(MinioClient.java:5149), io.minio.MinioClient.putObject(MinioClient.java:4337), io.minio.MinioClient.putObject(MinioClient.java:4183), FunctionalTest.removeIncompleteUploads_test(FunctionalTest.java:1719), FunctionalTest.runTests(FunctionalTest.java:2992), FunctionalTest.main(FunctionalTest.java:3118)]"
}

Executed 9 out of 14 tests successfully.
```

File: docs/config/README.md (hunk 1 above)
Comment (harshavardhana) [reply docs/config/README.md]:
```suggestion
Default trace is succinct only to indicate the API operations being called and the HTTP response status.
```

**Review by poornas — APPROVED** [review]

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-compression-dist-xl.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-xl.sh | :x: [more...](#ef4258a2-01d6-44f8-a98e-f705042628c7) |
| mint-large-bucket.sh | :x: [more...](#6790c8c5-fe33-4b75-9a3d-6f9181c2ee9d) |
| mint-worm.sh | :x: [more...](#a746872a-50b0-4c53-a98d-d7abea91fb2e) |
<a name="a746872a-50b0-4c53-a98d-d7abea91fb2e"></a>
### 7808-dfb70ef/mint-worm.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.57:32682
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            worm
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 75fae6f396c9:/mint/log /tmp/mint-logs'

(1/1) Running worm tests ... FAILED in 3 seconds
{
  "alert": "",
  "args": {
    "bucketName": "bucket-52frkn0s4iemqkxbzt7igrvqonlknf7kvxu1ao8diwndics50bonzwe",
    "expiry": 60000000000,
    "objectName": "testObject"
  },
  "duration": 3055,
  "error": "RequestError: send request failed\ncaused by: Put http://72.28.97.57:32682/bucket-52frkn0s4iemqkxbzt7igrvqonlknf7kvxu1ao8diwndics50bonzwe: dial tcp 72.28.97.57:32682: connect: no route to host",
  "function": "PutAndDelete",
  "message": "WORM_MODE ON  - CreateBucket Failed",
  "name": "test worm mode",
  "status": "FAIL"
}

Executed 0 out of 1 tests successfully.
```
<a name="6790c8c5-fe33-4b75-9a3d-6f9181c2ee9d"></a>
### 7808-dfb70ef/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.61:32280
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 5c30a0eb7ad0:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... done in 2 seconds
(2/14) Running aws-sdk-java tests ... done in 1 seconds
(3/14) Running aws-sdk-php tests ... done in 51 seconds
(4/14) Running aws-sdk-ruby tests ... done in 14 seconds
(5/14) Running awscli tests ... done in 2 minutes and 17 seconds
(6/14) Running healthcheck tests ... done in 1 seconds
(7/14) Running mc tests ... FAILED in 2 minutes and 8 seconds
{
  "name": "mc",
  "duration": "100113",
  "function": "test_mirror_list_objects_storage_class",
  "status": "FAIL",
  "error": "/mint/run/core/mc/mc --config-dir /tmp/.mc-25236 --quiet --no-color mirror --storage-class REDUCED_REDUNDANCY /mint/data myminio/mc-test-bucket-25683  >>> \n`/mint/data/datafile-1-MB` -> `myminio/mc-test-bucket-25683/datafile-1-MB`\n`/mint/data/datafile-0-b` -> `myminio/mc-test-bucket-25683/datafile-0-b`\n`/mint/data/datafile-1-kB` -> `myminio/mc-test-bucket-25683/datafile-1-kB`\n`/mint/data/datafile-1-b` -> `myminio/mc-test-bucket-25683/datafile-1-b`\n`/mint/data/datafile-10-kB` -> `myminio/mc-test-bucket-25683/datafile-10-kB`\n`/mint/data/datafile-11-MB` -> `myminio/mc-test-bucket-25683/datafile-11-MB`\n`/mint/data/datafile-10-MB` -> `myminio/mc-test-bucket-25683/datafile-10-MB`\n`/mint/data/datafile-1.03-MB` -> `myminio/mc-test-bucket-25683/datafile-1.03-MB`\n`/mint/data/datafile-100-kB` -> `myminio/mc-test-bucket-25683/datafile-100-kB`\n`/mint/data/datafile-129-MB` -> `myminio/mc-test-bucket-25683/datafile-129-MB`\n`/mint/data/datafile-5-MB` -> `myminio/mc-test-bucket-25683/datafile-5-MB`\n`/mint/data/datafile-33-kB` -> `myminio/mc-test-bucket-25683/datafile-33-kB`\n`/mint/data/datafile-6-MB` -> `myminio/mc-test-bucket-25683/datafile-6-MB`\n`/mint/data/datafile-5243880-b` -> `myminio/mc-test-bucket-25683/datafile-5243880-b`\n`/mint/data/datafile-65-MB` -> `myminio/mc-test-bucket-25683/datafile-65-MB`\nmc: <ERROR> Failed to copy `/mint/data/datafile-129-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-65-MB`. Please reduce your request\nmc: <ERROR> Failed to copy `/mint/data/datafile-10-MB`. Please reduce your request\nTotal: 233.17 MiB, Transferred: 532.58 MiB, Speed: 5.32 MiB/s"
}

Executed 6 out of 14 tests successfully.
```
<a name="ef4258a2-01d6-44f8-a98e-f705042628c7"></a>
### 7808-dfb70ef/mint-xl.sh.log:
```
Running with
SERVER_ENDPOINT:      72.28.97.51:30864
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp e1770923200a:/mint/log /tmp/mint-logs'

(1/14) Running aws-sdk-go tests ... FAILED in 6 minutes and 0 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-k04p1n9g6ii5ys",
    "expiry": 60000000000,
    "objectName": "presignedTest"
  },
  "duration": 360014,
  "error": "XMinioServerNotInitialized: Server not initialized, please try again.\n\tstatus code: 503, request id: 15A9AE5399A3C8A0, host id: ",
  "function": "PresignedPut",
  "message": "AWS SDK Go CreateBucket Failed",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 14 tests successfully.
```

**Review by kannappanr — APPROVED** [review]

**Pull request merged**